install/install.py

```diff
@@ -228,14 +252,32 @@ def danger(cls, string):
 
 
 if __name__ == "__main__":
+    root_dir = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
+    pid = os.getpid()
+
+    # custom scripts
```

Comment:
# `Path to custom scripts for tasks such as setting up/ running PostgreSQL db, run arachni, nikto, wapiti etc.`